### DISCUSSION TITLE
Is ARC Support enabled by default?

### DISCUSSION THREAD

**Discussion by mattfox27** [Jyers/Marlin#1205] [General Discussion]
I'm trying to use arc welder at octoprint is arc support enabled by default It looks like it is but just want to double check?

Ty

**Comment by Cal-LF** [comment]
Arc support is activated by default.
Config_adv.h 

// G2/G3 Arc Support
//
#define ARC_SUPPORT                 // Disable this feature to save ~3226 bytes
#if ENABLED(ARC_SUPPORT)
  #define MM_PER_ARC_SEGMENT      1 // (mm) Length (or minimum length) of each arc segment
  //#define ARC_SEGMENTS_PER_R    1 // Max segment length, MM_PER = Min
  #define MIN_ARC_SEGMENTS       24 // Minimum number of segments in a complete circle
  //#define ARC_SEGMENTS_PER_SEC 50 // Use feedrate to choose segment length (with MM_PER_ARC_SEGMENT as the minimum)
  #define N_ARC_CORRECTION       25 // Number of interpolated segments between corrections
  #define ARC_P_CIRCLES           // Enable the 'P' parameter to specify complete circles
  //#define CNC_WORKSPACE_PLANES    // Allow G2/G3 to operate in XY, ZX, or YZ planes
  //#define SF_ARC_FIX              // Enable only if using SkeinForge with "Arc Point" fillet procedure
#endif

**Comment by mattfox27** [comment]
Do I need to enable it?  Or is in enabled already?

**Reply by Cal-LF** [reply]
@mattfox27 Already enabled..............
I have tried Arc Welder and it does seem to work on cylinders.

**Comment by cbarth3** [comment]
Seeing this on the latest version of ARC Welder

Your firmware may have issues running gcode produced by Arc Welder.
Name:
Marlin
Version:
Marlin-JyersUI
Supported:
No
Arcs Supported:
Yes
3D-Arcs Supported:
Unknown
Arcs Enabled:
Yes
Warnings:

    Your firmware version indicates that it is not supported, but arcs appear to be supported and enabled. Use with caution.
    Cannot determine if 3D Arc commands are supported (for use with vase mode). Since 3D arcs are currently disabled, this should be OK.
